.ci/azure/pipelines/obs-packaging-ci.yml

```diff
@@ -11,9 +11,37 @@ pool:
   vmImage: 'ubuntu-latest'
 
 steps:
-- script: echo setup
-  displayName: 'setup CI'
+# azure docker installation task
+# https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/docker-installer?view=azure-devops 
+- task: DockerInstaller@0
+  displayName: 'Install Docker'
 
-- script: |
-    echo This is an initial CI job
-  displayName: 'Run CI'
+- bash: |
+   sudo apt-get update
+   sudo apt-get install -y git
+   git config --global user.email "azure-pipeline@kata.io"
+   git config --global user.name "azure-pipeline"
+  displayName: 'Setup'
+
+- bash: |
+   # This is set by azure, but we use the same variable
+   export AGENT_VERSION=""
+   script -qefc bash -c '.ci/packaging/request_to_obs.sh'
+   # Kata branch to build
+  displayName: 'Request build to OBS'
+
+  env:
+    CI: true
+    BRANCH: $(system.pullRequest.targetBranch)
+    OBS_USER: $(OBS_USER)
+    OBS_PASS: $(OBS_PASS)
+    GITHUB_PR: $(system.pullRequest.pullRequestNumber)
+
+- bash: |
+   echo "Wait for packages from ${OBS_SUBPROJECT}"
+   [ "${OBS_SUBPROJECT}" != "" ] || exit 1
```

Comment:
maybe an `|| echo "No OBS_SUBPROJECT set"` or similar to make it more obvious in the logs why it quit.